src/OpenLoco/ScenarioManager.h

```diff
@@ -1,8 +1,49 @@
 #pragma once
 
+#include <cstddef>
 #include <cstdint>
 
 namespace OpenLoco::scenariomgr
 {
+    enum ScenarioIndexFlags : uint8_t
+    {
+        flag_1 = 1 << 0,
+        completed = 1 << 1,
+        hasPreviewImage = 1 << 2,
+    };
+
+    struct ScenarioIndexEntry
+    {
+        char filename[0x100];           // 0x000
+        uint8_t category;               // 0x100
+        uint8_t numCompetingCompanies;  // 0x101
+        uint8_t competingCompanyDelay;  // 0x102
+        uint8_t pad_103[0x120 - 0x103]; // 0x103
+        uint16_t startYear;             // 0x120
+        uint16_t completedMonths;       // 0x122
+        char scenarioName[0x40];        // 0x124
+        char description[0x100];        // 0x164
+        uint32_t flags;                 // 0x264
+        char highscoreName[0x100];      // 0x268
+        char objective[0x100];          // 0x368
+        uint32_t currentObjectPtr;      // 0x468
```

Comment:
This seems like it should be an `ObjectManager::header` instead (as used in `S5.h`)